### ISSUE TITLE
Insert page between split pages

### ISSUE THREAD

**Issue by lifepillar** [4lex4/scantailor-advanced#157]
How do you move a page between a split page?

I have scanned a book where pages 3 and 4 are missing. So, I have one image with page 2 and 5 side by side. Later, I have found and scanned pages 3 and 4 (as two separate scans). Now I have three images (page 2&5, page 3, page 4), which I import into STA. “Split Pages” splits pages 2 and 5 correctly, but page 3 and page 4 appear after page 5. Can I move page 3 and 4 between page 2 and 5? “Insert before…”/“Insert after…” ask me for an image file, and I have a single file for pages 2&5…

**Comment by Piolie** [comment]
I think STA does not support inserting images in between split pages. Not the ideal solution, but you could manually split page 2&5 and reimport 2, 3, 4 and 5 in the correct order.

**Comment by mara004** [comment]
Yeah, I don't think there currently is a better way. It would be nice if STA had features to rearrange pages better...

**Comment by mara004** [comment]
Yeah, I don't think there currently is a better way. It would be nice if STA had features to rearrange pages...
